src/sass/edu-benefits.scss

```diff
@@ -25,3 +25,13 @@ select {
 legend {
   @extend h4;
 }
+
+.form-indent {
```

Comment:
Hm I would check USWDS to see what kinds of styles they have for these things. I think they have something for the box at least.